### ISSUE TITLE
Edit on Github links are dead

### ISSUE THREAD

**Issue by juniperbelmont** [Polymer/docs#1627]
All "Edit on Github" links point to non-existent locations.
They're pointing to a repository that's inaccessible: Polymer/docs2

Eg. [The Polymer Devguide](https://www.polymer-project.org/1.0/docs/devguide/feature-overview) "Edit on Github" link points to https://github.com/Polymer/docs2/edit/master/app/1.0/docs/devguide/feature-overview.md which gives a Github 404 error.

**Comment by notwaldorf** [comment]
Eeek, sorry about that! The `docs2` repo is currently still private -- we're cleaning up some stuff before we can make it public (trying to merge the history and issues from this repo), and after that those links should start working.

Sorry about the inconvenience in the meantime!

/cc @arthurevans

**Comment by notwaldorf** [comment]
closing as duplicate of https://github.com/Polymer/docs/issues/1622
